core/templates/dev/head/pages/email_dashboard/email-dashboard.mainpage.html

```diff
@@ -83,6 +83,7 @@
 
 {% block footer_js %}
   {{ super() }}
-  <script src="/templates/dev/head/pages/email_dashboard/EmailDashboard.js"></script>
+  <script src="/templates/dev/head/pages/email_dashboard/email-dashboard.module.js"></script>
```

Comment:
Are these scripts required here?

Reply:
I am not sure, (comment link https://github.com/oppia/oppia/pull/6524#discussion_r278119285). I will try to remove them and push the changes if those pass tests then ill keep them removed. 
And also the tests were failing when I did not include them in base or admin so I am not sure what is happening. I will try to remove them and then push changes and if it doesnt work then Ill revert it.

Reply:
done